static_report/src/components/shared/DescriptionTooltip.tsx

```diff
@@ -0,0 +1,33 @@
+import { Text, Tooltip } from '@chakra-ui/react';
+
+type Props = { label: string; via: string };
+function DescriptionLabel({ label, via }: Props) {
+  return (
+    <Text>
+      {label}
+      <Text as="i" color="gray.400">
+        {` - via ${via}`}
+      </Text>
+    </Text>
+  );
+}
+
+export function DescriptionTooltip({ description, children, ...props }) {
```

Comment:
I don't think the `Description*` prefix is necessary and is a bit redundant to the prop `description`. A tooltip imho should be an agnostic, shared UI Component for presentation.

Secondly, I think the regex should be a Props, so that we aren't locked into the `- via ` prefix

Naming wise, i'd call this `EnrichedTooltip`, as we are adding to the existing one, and we can still use it in non-description contexts

Reply:
Also, Please Add comment block describing this tooltip :)

Reply:
Is it a good idea to separate `description` and `via` in run.json file? Thus we don't need regex logic in frontend, just pass `label` and `via` to `EnrichedTooltip`.

Reply:
Renamed to `SRTooltip` for showing specified description in SingleReport